ntp/responder/server/server.go

```diff
@@ -147,14 +148,14 @@ func (s *Server) startListener(ip net.IP, port int) {
 		log.Fatalf("enabling timestamp error: %s", err)
 	}
 
-	buf := make([]byte, timestamp.PayloadSizeBytes)
-	oob := make([]byte, timestamp.ControlSizeBytes)
-
 	err = unix.SetNonblock(connFd, false)
 	if err != nil {
 		log.Fatalf("Failed to set socket to blocking: %s", err)
 	}
 
+	buf := make([]byte, timestamp.PayloadSizeBytes)
```

Comment:
While on it, do we want an NTP specific payload size?

Reply:
Dunno, PayloadSizeBytes is 128, and typical NTP packet is 48 bytes. But we have only one buffer, so no memory wins here. The only potential problem is if PayloadSizeBytes gets reduced to less than 48 bytes, which is not gonna happen with PTP packets being bigger.